R/dependencies.R

```diff
@@ -3,47 +3,53 @@
 
 #' Find R Package Dependencies in a Project
 #'
-#' Find \R packages used within a project.
-#'
+#' @description
 #' `dependencies()` will crawl files within your project, looking for \R files
 #' and the packages used within those \R files. This is done primarily by
-#' parsing the code and looking for calls of the form:
-#'
-#' - `library(package)`
-#' - `require(package)`
-#' - `requireNamespace("package")`
-#' - `package::method()`
+#' parsing the code and looking for calls of the form `library(package)`,
```

Comment:
Here I collapsed it because I don't think the exact details are likely to be that important for most users; `dependencies()` will just magically work. This also gets more content "above the fold", e.g. compare to https://rstudio.github.io/renv/reference/dependencies.html where you have to scroll past Arguments and Value until you get to the basic operation of the function.